src/api/category.js

```diff
@@ -1,10 +1,15 @@
-module.exports.register = (server) => {
+function register (server) {
```

Comment:
any reason why it moved away from double-arrow notation here?

Reply:
Oh none, actually! I will switch it to arrow function 👍